package.json

```diff
@@ -26,23 +26,29 @@
   },
   "homepage": "https://github.com/Kvoti/redux-rest",
   "devDependencies": {
-    "babel": "^5.5.8",
-    "babel-core": "5.6.15",
+    "babel-core": "^6.0.0",
     "babel-eslint": "^4.0.5",
-    "babel-loader": "^5.1.4",
+    "babel-loader": "^6.1.0",
+    "babel-plugin-transform-class-properties": "^6.6.0",
+    "babel-plugin-transform-decorators": "^6.6.0",
+    "babel-plugin-transform-object-rest-spread": "^6.5.0",
+    "babel-preset-es2015": "^6.6.0",
     "eslint": "^0.24.1",
     "eslint-config-airbnb": "0.0.6",
     "eslint-plugin-react": "^3.1.0",
     "expect": "^1.8.0",
     "isparta": "^3.0.3",
     "mocha": "^2.2.5",
+    "nock": "^2.9.1",
     "react": "^0.13.0",
+    "redux": ">=1.0.0-rc",
```

Comment:
This should be a peer dependency (my bad) as it's not a requirement of redux-rest but redux-rest is only compatible with version 1.0.0 and above.